extension/secrets.go

```diff
@@ -43,17 +45,17 @@ func readSecretIdsFromEnvironmentWhenStartsWithSecret() []string {
 	return secretIds
 }
 
-func getSecretValuesFromListOfSecretIds(secretIds []string) []Secret {
-	var secrets []Secret
+func getSecretValuesFromListOfSecretIds(secretIds []string, secretsChannel chan Secret) {
 	cfg, err := config.LoadDefaultConfig(context.TODO())
 	checkError(err)
 	client := secretsmanager.NewFromConfig(cfg)
 
 	for _, secretId := range secretIds {
+		log.Print("Fetch secret: "+secretId+" on ", time.Now().UnixMilli())
 		secret := getSecretValue(client, secretId)
```

Comment:
I think the go command should be here so that all the getSecret calls run in parralel.